### PR TITLE
Deployment script and dependencies

### PR DESCRIPTION
**Pull request by MagoDelBlocco**
Added a script for deploying directly on the GCP. It might require gcloud authentication before running the script.
How to use: `./deploy.sh`.

Changed the deprecated dependency "maven-appengine" to "google.cloud.maven-appengine".
Added .xml file in WEB-INF for appengine to pick up and run.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+gcloud config set project llvm-build-dashboard &&
+mvn package com.google.cloud.tools:appengine-maven-plugin:deploy
```
Comment (samuelsogbesan) [inline deploy.sh:3]: (An issue needs to be created to create the app.yaml file)

**Review by samuelsogbesan — APPROVED** [review]
LGTM, note made to open an issue about the app.yaml file.

File: deploy.sh (hunk 1 above)
Comment (MagoDelBlocco) [reply deploy.sh:3]: #108

**Review by krasimir-google — APPROVED** [review]
LGTM

**Pull request merged**